frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -42,7 +67,14 @@ export const supportLogic = kea<supportLogicType>([
             },
         },
     })),
-    listeners(({}) => ({
+    listeners(({ actions }) => ({
+        openSupportForm: async ({ kind, target_area }) => {
+            actions.resetSendSupportRequest({
+                kind: kind ? kind : null,
+                target_area: target_area ? target_area : null,
```

Comment:
Since `kind` and `target_area` can explicitly be null and shouldn't ever be undefined you can just say `kind: kind` (or just `kind` if you want to use the shorthand notation).